### PR TITLE
MAT-7790: Function Builder defaults

### PR DESCRIPTION
**Pull request by chubert-sb**
## MADiE PR

Jira Ticket: [MAT-7790](https://jira.cms.gov/browse/MAT-7790)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/CqlBuilderPanel/functionsSection/Functions.scss [hunk 1]
```diff
@@ -67,6 +67,10 @@
     > .cancel-button {
       margin-top: 1rem;
       margin-right: 32px;
+      display: none;
```
Comment (mcmcphillips) [inline src/CqlBuilderPanel/functionsSection/Functions.scss]: Although we've confirmed that this display none rule isn't hitting, we should probably not include it in the pr.

Comment (chubert-sb) [reply src/CqlBuilderPanel/functionsSection/Functions.scss]: whoops forgot to remove that. thanks!

File: src/validations/FunctionSectionSchemaValidator.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import * as Yup from "yup";
+
+export const FunctionSectionSchemaValidator = Yup.object().shape({
+  functionName: Yup.string()
+    .required("Function name is required")
+    .matches(/^[a-zA-Z0-9]*$/, "No spaces or special characters are allowed"),
```
Comment (adongare) [inline src/validations/FunctionSectionSchemaValidator.ts]: I think spaces are allowed. This needs to be confirmed with Stan/POs.

Comment (chubert-sb) [reply src/validations/FunctionSectionSchemaValidator.ts]: removed the matches criteria

**Review by adongare — APPROVED** [review]

**Pull request merged**